src/components/Exchange/SwapBox.js

```diff
@@ -212,9 +209,8 @@ export default function SwapBox(props) {
 
   const [ordersToaOpen, setOrdersToaOpen] = useState(false);
 
-  let [orderOption, setOrderOption] = useLocalStorageSerializeKey([chainId, "Order-option"], MARKET);
   if (!flagOrdersEnabled) {
-    orderOption = MARKET;
+    setOrderOption(MARKET);
```

Comment:
it's better to move it inside useEffect and check that option is not MARKET already